cpp/src/io/parquet/reader_impl_preprocess.cu

```diff
@@ -307,8 +379,13 @@ int decode_page_headers(cudf::detail::hostdevice_vector<ColumnChunkDesc>& chunks
   DecodePageHeaders(chunks.device_ptr(), chunks.size(), error_code.data(), stream);
 
   if (error_code.value() != 0) {
-    // TODO(ets): if an unsupported encoding was detected, do extra work to figure out which one
-    CUDF_FAIL("Parquet header parsing failed with code(s)" + error_code.str());
+    if (BitAnd(error_code.value(), decode_error::UNSUPPORTED_ENCODING) == 0x4) {
```

Comment:
I don't think we need to compare with exact (hardcoded value) here
```suggestion
    if (BitAnd(error_code.value(), decode_error::UNSUPPORTED_ENCODING) != 0) {
```

Reply:
RIP, I did this to test something and forgot to change it back. Mb